### DISCUSSION TITLE
New Kindle

### DISCUSSION THREAD

**Discussion by martinmystere** [apprenticeharper/DeDRM_tools#2079] [Q&A]
Hello, 
i bought a new kindle (i will arrive in a couple of days) and sold the old one... after changing the device SN in the plugin settings do i need to re-import every book in the library?

Thank you!

Luca

**Comment by ElleKayEm** [comment]
Only books you haven't previously imported and removed DRM from already.

**Reply by martinmystere** [reply]
Ok thanks!
I thought the books would have been "binded" to the declared SN even though DRM-free.

L

**Reply by ElleKayEm** [reply]
Exactly the opposite, removing the DRM "unbinds" them from that device.

**Reply by martinmystere** [reply]
Oh I see!! Thank you!!
L
